api/desecapi/models.py

```diff
@@ -169,18 +178,26 @@ class Token(rest_framework.authtoken.models.Token):
     name = models.CharField("Name", max_length=64, default="")
     user_specific_id = models.BigIntegerField("User-Specific ID")
 
+    class Meta:
+        abstract = False
```

Comment:
no-op?

Reply:
that was just reordering because we usually have the `Meta` class before the method declarations. I can undo the change, or put it into a separate commit. Opinions?

Reply:
I'd put it in a separate commit, but I won't be dogmatic about it